src/main/Mainpage.hpp

```diff
@@ -38,6 +36,6 @@
  * As you prepare to develop code for Clio, please be sure you are aware of our current
  * <A HREF="https://github.com/XRPLF/clio/blob/develop/CONTRIBUTING.md">Contribution guidelines</A>.
  *
- * Read `rpc/README.md` carefully to know more about writing your own handlers for
+ * Read <A HREF="https://github.com/XRPLF/clio/blob/develop/src/rpc/README.md">rpc/README.md</A> carefully to know more about writing your own handlers for
```

Comment:
This now always will open latest `develop`, which may be ok. I think it would be better if we could open the README.md from the same doc (generated by doxygen or on github). I'm not sure if that's even possible. Maybe with some relative links 🤷‍♂️ ❓

Reply:
Note that the link above that leads to `CONTRIBUTING.md` is always opened from latest develop and that makes sense as we want new developers to always read the latest guide 👍 I'm not sure same applies for other READMEs though..